api/controllers/mypageController.js

```diff
@@ -0,0 +1,61 @@
+const mypageService = require('../services/mypageService');
+const { catchAsync } = require('../middlewares/error');
+
+const getSellerInfo = catchAsync(async (req, res) => {
+  const userId = req.userId.userId.id;
```

Comment:
코드가 아래와 같이 되어야 id를 가져올 수 있을 것 같은데, 확인해 주시고, 만약 그게 아니라면, 아래와 같이 썼을 때 userId를 가져올 수 있게 수정해 주세요.
불필요한 property가 너무 반복적으로 사용 됩니다.
```suggestion
  const userId = req.userId.id;
```